packages/contracts/test/contracts/OVM/precompiles/OVM_SequencerEntrypoint.spec.ts

```diff
@@ -94,141 +87,69 @@ describe('OVM_SequencerEntrypoint', () => {
     })
 
-    for (let i = 0; i < 3; i += 2) {
-      it(`should call ovmCreateEOA when tx type is ${i} and ovmEXTCODESIZE returns 0`, async () => {
-        let firstCheck = true
-        Mock__OVM_ExecutionManager.smocked.ovmEXTCODESIZE.will.return.with(
-          () => {
-            if (firstCheck) {
-              firstCheck = false
-              return 0
-            } else {
-              return 1
-            }
-          }
-        )
-        const calldata = await encodeSequencerCalldata(
-          wallet,
-          DEFAULT_EIP155_TX,
-          i
-        )
-        await Helper_PredeployCaller.callPredeploy(
-          OVM_SequencerEntrypoint.address,
-          calldata
-        )
-        const call: any =
-          Mock__OVM_ExecutionManager.smocked.ovmCREATEEOA.calls[0]
-        const eoaAddress = ethers.utils.recoverAddress(call._messageHash, {
-          v: call._v + 27,
-          r: call._r,
-          s: call._s,
-        })
-        expect(eoaAddress).to.equal(await wallet.getAddress())
+    it(`should call ovmCreateEOA when ovmEXTCODESIZE returns 0`, async () => {
+      let isFirstCheck = true
+      Mock__OVM_ExecutionManager.smocked.ovmEXTCODESIZE.will.return.with(() => {
+        if (isFirstCheck) {
+          isFirstCheck = false
+          return 0
+        } else {
+          return 1
+        }
       })
-    }
 
-    it('should submit ETHSignedTypedData if TransactionType is 2', async () => {
-      const calldata = await encodeSequencerCalldata(
-        wallet,
-        DEFAULT_EIP155_TX,
-        2
-      )
+      const transaction = DEFAULT_EIP155_TX
+      const encodedTransaction = await wallet.signTransaction(transaction)
+
       await Helper_PredeployCaller.callPredeploy(
         OVM_SequencerEntrypoint.address,
-        calldata
-      )
-
-      const encodedTx = serializeEthSignTransaction(DEFAULT_EIP155_TX)
-      const sig = await signEthSignMessage(wallet, DEFAULT_EIP155_TX)
-
-      const expectedEOACalldata = iOVM_ECDSAContractAccount.encodeFunctionData(
-        'execute',
-        [
-          encodedTx,
-          1, //isEthSignedMessage
-          `0x${sig.v}`, //v
-          `0x${sig.r}`, //r
-          `0x${sig.s}`, //s
-        ]
+        encodedTransaction
       )
-      const ovmCALL: any = Mock__OVM_ExecutionManager.smocked.ovmCALL.calls[0]
-      expect(ovmCALL._address).to.equal(await wallet.getAddress())
-      expect(ovmCALL._calldata).to.equal(expectedEOACalldata)
-    })
 
-    it('should revert if TransactionType is >2', async () => {
-      const calldata = '0x03'
-      await expect(
-        Helper_PredeployCaller.callPredeploy(
-          OVM_SequencerEntrypoint.address,
-          calldata
-        )
-      ).to.be.reverted
-    })
+      const call: any = Mock__OVM_ExecutionManager.smocked.ovmCREATEEOA.calls[0]
+      const eoaAddress = ethers.utils.recoverAddress(call._messageHash, {
+        v: call._v + 27,
```

Comment:
todo(karl): look into why this `v + 27` works when the chain ID of the `DEFAULT_EIP155_TX` is 420

Reply:
What do you mean by this?